io.github.webcamoid.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this file isn't referenced anywhere, let alone installed.

Reply:
To clarify: you need to add `install -Dm644 $FLATPAK_ID.metainfo.xml /app/share/metainfo/$FLATPAK_ID.metainfo.xml` to `build-commands` of some module, and reference this file in the `sources` array. This is the last change needed to make build, probably.